Tests/id_set.json

```diff
@@ -12757,7 +12760,7 @@
                     "DeleteContext"
                 ], 
                 "command_to_integration": {
-                    "whois": "whois"
+                    "whois": "Whois"
```

Comment:
Whois? why is this changed?

Reply:
Idk, I just ran the automated script. Should I change it back?

Reply:
Compare it with master and take what's in there

Reply:
It's the correct one...